preset/src/shortcuts/button.ts

```diff
@@ -6,37 +6,37 @@ export const buttonShortcuts: Record<string, string> = {
 	'k-button--circle': '!rounded-full p-2',
 
 	'k-button--primary':
-		'bg-ikun-primary-100 text-ikun-primary border-ikun-primary-100',
+		'bg-ikun-primary-100 !text-ikun-primary border-ikun-primary-100',
```

Comment:
Why use `important`?

Reply:
I noticed that the styles are being overridden in VitePress when the button tag is set to 'a'.

![image](https://github.com/ikun-svelte/ikun-ui/assets/49926816/9a3e5ba1-d91b-4f43-9977-c2cc50229277)